diff --git a/node_modules/@neondatabase/serverless/index.mjs b/node_modules/@neondatabase/serverless/index.mjs
index e1263d6..6ca8fd3 100644
--- a/node_modules/@neondatabase/serverless/index.mjs
+++ b/node_modules/@neondatabase/serverless/index.mjs
@@ -1,14 +1,15 @@
 import mod from "./index.js";
 
-export default mod;
-export const Client = mod.Client;
-export const ClientBase = mod.ClientBase;
-export const Connection = mod.Connection;
-export const DatabaseError = mod.DatabaseError;
-export const NeonDbError = mod.NeonDbError;
-export const Pool = mod.Pool;
-export const Query = mod.Query;
-export const defaults = mod.defaults;
-export const neon = mod.neon;
-export const neonConfig = mod.neonConfig;
-export const types = mod.types;
+export {
+  Client,
+  ClientBase,
+  Connection,
+  DatabaseError,
+  NeonDbError,
+  Pool,
+  Query,
+  defaults,
+  neon,
+  neonConfig,
+  types,
+} from './index.js';
\ No newline at end of file
